tasks/install.yml

```diff
@@ -140,6 +196,23 @@
     dest: /etc/tomcat/Catalina/localhost/rest.xml
     backup: yes
 
+- name: config XMLWorflow
```

Comment:
I suspect that we'll want versioned copies of these files located in individual DSpace site git repositories for Commons and SHAREOK, but it looks like we still need to make the shift from building a default DSPace base by running the installer to building DSpace based on a site specific repo.  Maybe we merge this in, and then the next step is going through and figure out what pieces of the config should live in Ansible and what pieces should live in the site git repos?